packages/api/src/page/page.module.ts

```diff
@@ -1,11 +1,12 @@
 import { Module } from '@nestjs/common'
 import { TypeOrmModule } from '@nestjs/typeorm'
-import { Page, PageRepository, PageService } from '@webok/core/lib/page'
+import { Page } from '@webok/core/lib/page'
+import { PageRepository, PageService } from '@webok/services/lib/page'
 import { PageController } from './page.controller'
 
 @Module({
   imports: [TypeOrmModule.forFeature([Page, PageRepository])],
-  providers: [PageService],
+  providers: [{ provide: 'IPageService', useClass: PageService }],
```

Comment:
Need to use string here because interface is not value type, see https://github.com/nestjs/nest/issues/43.